hmmlearn/utils.py

```diff
@@ -60,3 +60,9 @@ def iter_from_X_lengths(X, lengths):
 
         for i in range(len(lengths)):
             yield start[i], end[i]
+
+def prob_log(prob):
```

Comment:
we can pick up log_mask_zero back if you think it's acceptable, :)
let me refine the patch to request another round of comment.